src/components/WalletConnect.vue

```diff
@@ -52,23 +52,45 @@
 		<a-row class="wallet">
 			<a-card size="small">
 				<a-row align="middle">
-					<a-col :span="20">
-						<a-descriptions :column="1" size="small" class="ellipsis">
-							<a-descriptions-item label="Wallet">{{
-								walletAddress
-							}}</a-descriptions-item>
-							<a-descriptions-item label="Selected Wallet">
-								{{ selectedWallet }}
-							</a-descriptions-item>
-						</a-descriptions>
-					</a-col>
-					<a-col style="margin: auto">
-						<a-button type="primary" @click="handleLogOut">
-							<template #icon>
-								<LogoutOutlined />
-							</template>
-						</a-button>
-					</a-col>
+					<span>
+						<a-typography-text strong>Address: </a-typography-text>
+						<a-typography-link :copyable="{ text: walletAddress }">{{
+							walletAddress
+						}}</a-typography-link>
+					</span>
+					<a-row type="flex" style="width: 100%">
+						<a-col :flex="1">
+							<span style="margin-top: 5px">
+								<a-typography-text strong>Wallet: </a-typography-text>
+								<a-typography-link>{{ selectedWallet }}</a-typography-link>
+							</span>
+						</a-col>
+						<a-col>
+							<a-dropdown>
+								<template #overlay>
+									<a-menu @click="handleAccountSwitch">
+										<a-menu-item v-for="addr in walletAddresses" :key="addr">
```

Comment:
What happened if `walletAddresses` is empty? Should we handle this case

Reply:
Already handled.